src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -153,13 +154,19 @@ private BoundExpression BindInterpolatedString(InterpolatedStringExpressionSynta
                         case SyntaxKind.InterpolatedStringText:
                             {
                                 var text = ((InterpolatedStringTextSyntax)content).TextToken.ValueText;
-                                builder.Add(new BoundLiteral(content, ConstantValue.Create(text, SpecialType.System_String), stringType));
+                                // Raw string literals have no escapes.  So there is no need to manipulate their value texts.
+                                if (!isRawInterpolatedString)
+                                {
+                                    text = ConstantValueUtils.UnescapeInterpolatedStringLiteral(text);
+                                }
+
+                                var constantValue = ConstantValue.Create(text, SpecialType.System_String);
+                                builder.Add(new BoundLiteral(content, constantValue, stringType));
```

Comment:
the core cahnge here is that we always make the ConstantValue have the right actual text content.  THe unescaping only happens here when we make the boundliteral (and not for raw strings).  From this point on, nothing should touch this text.